### PR TITLE
minor grammar cleanup

### PR DESCRIPTION
**Pull request by chaals**
***
<a href="https://pr-preview.s3.amazonaws.com/chaals/privacy-principles/pull/407.html" title="Last updated on Feb 28, 2024, 5:21 PM UTC (1f1c723)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/407/064c571...chaals:1f1c723.html" title="Last updated on Feb 28, 2024, 5:21 PM UTC (1f1c723)">Diff</a>

### CODE REVIEW

**Review by darobin — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -632,7 +632,7 @@
 need of protection.
 
 <dfn data-lt="governance">Data governance</dfn> is the system of principles that regulate [=information flows=].
-[=Data governance=] determines how
+[=Data governance=] determines
 which [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
```
Comment (jyasskin) [inline index.html:636]: The "how" here was actually doing work, but I agree with @chaals that it was confusing. The original meaning was that "Data governance determines which actors can collect data, **how those actors can collect the data**, and how they're allowed to process it." This change loses that second piece. Do we want to keep it?

Comment (jyasskin) [inline index.html:636]:
```suggestion
[=Data governance=] determines
which [=actors=] can collect [=data=], what data they can collect, how they can collect it, and how they may, must, or must not [=process=] it
```

Comment (jyasskin) [reply index.html:636]:
```suggestion
[=Data governance=] determines
which [=actors=] can collect [=data=], what data they can collect, how they can collect it, and how they can [=process=] it
```

**Review by torgo — APPROVED** [review]
lgtm

**Pull request merged**